exporters/trace/jaeger/jaeger_test.go

```diff
@@ -37,26 +37,34 @@ import (
 	sdktrace "go.opentelemetry.io/otel/sdk/trace"
 )
 
-func TestNewExporterPipelineWithRegistration(t *testing.T) {
-	tp, fn, err := NewExportPipeline(
+func TestInstallNewPipeline(t *testing.T) {
+	fn, err := InstallNewPipeline(
 		WithCollectorEndpoint("http://localhost:14268/api/traces"),
-		RegisterAsGlobal(),
 	)
 	defer fn()
 	assert.NoError(t, err)
-	assert.Same(t, tp, global.TraceProvider())
+	assert.IsType(t, &sdktrace.Provider{}, global.TraceProvider())
 }
 
-func TestNewExporterPipelineWithoutRegistration(t *testing.T) {
+func TestInstallNewPipeline_WithDisabled(t *testing.T) {
```

Comment:
I'm in favor of sticking with Go conventions of mixed capitalization naming and not introducing underscores.

If we want to delineate the testing it might be useful to abstract the "`TestInstallNewPipeline`" part of the test from the different cases and use `t.Run` with separate broken out functions.

Reply:
Honestly I'm used to both ways of test naming, even though I'm aware of the mixed caps convention. I have no strong feeling either way, so breaking it into sub-tests with `t.Run` is entirely fine with me.

Reply:
please no underscores, thanks <3

Reply:
@lizthegrey duly noted :wink: